aten/src/ATen/native/native_functions.yaml

```diff
@@ -753,6 +753,10 @@
   dispatch:
     QuantizedCPU: empty_affine_quantized_cpu
 
+- func: _empty_perchannel_affine_quantized_like(Tensor self, Tensor zero_points, int[] size, int[] axis, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None, MemoryFormat? memory_format=contiguous_format) -> Tensor
```

Comment:
I think you can probably create a _empty_affine_quantized tensor and use: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L3019 to change the quantizer as a work around.

Reply:
The link points to an empty line. I didn't get what you mean by change the quantizer as a work around?